nss-day-cohort-41/tabloid-fullstack-first-amendment-dancers-andy
# Ba 2nd posts

# Description
Completed post functionality as well as basic style layout.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
# Testing Instructions
Pull down this branch and run it.  Verify that all previously working features are still functional.  Verify that the post list list displays in a table format.  Verify that you can edit and delete posts that belong to the user you are logged in as, and that you cannot delete posts that do not belong to that user.  
# Checklist:
